bin/run-suite

```diff
@@ -6,6 +6,54 @@ import subprocess
 import sys
 import copy
 import timeit
+import io
+import argparse
+
+app_list = [
```

Comment:
Would it make sense to create a more focused list/a different profile for the benchmarks we're currently interested in?

Reply:
Added the option `--no-bandwidth-test` which excludes the bandwidth we don't care about